src/main/java/dev/pgm/events/ready/ReadyListener.java

```diff
@@ -58,17 +62,16 @@ public void onLeave(PlayerLeaveMatchEvent event) {
 
   @EventHandler(priority = EventPriority.MONITOR)
   public void onPartyChange(PlayerPartyChangeEvent event) {
-    if (!AppData.readyReminders()) {
+    if (!AppData.readyReminders()
```

Comment:
Instead of checking the match phase could we use `readyManager.canReady` for a better check?